build/Makefile

```diff
@@ -856,3 +856,10 @@ test-gen-crd-client:
 		echo "$$diff_output"; \
 	fi
 
+update-agones: $(ensure-build-image)
+	docker run -it --rm \
+		$(common_mounts) \
+		-w $(workdir_path) \
```

Comment:
```suggestion
```

Given that this is all helm commands - probably doesn't need to be in a specific work dir?

Reply:
removed this line